tracks/golang/_posts/2024-01-05-golang-week-2.md

```diff
@@ -59,4 +59,7 @@ feature_text: |
 [Resourse Link](https://youtu.be/gZ-wl1no_3s?feature=shared)
 
 #### Locking variables in golang using mutex
-[Resourse Link](https://youtu.be/v7DKtAluvYA?feature=shared)
\ No newline at end of file
+[Resourse Link](https://youtu.be/v7DKtAluvYA?feature=shared)
+
+#### Week 2 Assignment 
+[Resource Link](https://docs.google.com/document/d/11wZ0X2AryTioB5hp0kgJDEfblBdfF8ceh2i2ENISunQ/edit?usp=sharing)
```

Comment:
Replace this from a resource link to a embedded PDF. 
Use the follow tag 
```<object data="/uploads/golang-data/Golang-Week2-Assignment.pdf" width="1000" height="1000" type='application/pdf'></object>```

Export the document as a PDF and upload it in the uploads folder, create a folder for golang and keep it there.